client/my-sites/site-settings/form-writing.jsx

```diff
@@ -104,7 +104,7 @@ class SiteSettingsFormWriting extends Component {
 
 							<CustomContentTypes
 								onSubmitForm={ this.props.handleSubmitForm }
-								handleToggle={ handleToggle }
+								handleToggle={ handleAutosavingToggle }
```

Comment:
I recommend that we also alter the passed prop to be named `handleAutosavingToggle` to avoid confusion. If I see `handleToggle` in `CustomContentTypes`, I wouldn't expect it to be `handleAutosavingToggle`.